Stopwatch-JMBR/styles.css

```
@@ -0,0 +1,194 @@
+.hidden {
+  display: none;
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)